### PR TITLE
Add language ca and es

### PR DESCRIPTION
**Pull request by psd24**

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -135,8 +135,8 @@ t:
         content: "Le centre national d'intelligence artificielle chilien avec qui on travaille pour notre premier déploiement à l’étranger"
       partner4:
         title: "ADF Subirats"
-        url: "https://sfadf.org/adf/subirats//"
-        logo: "https://sfadf.org/wp-content/themes/sfadf/img/logo-header.jpg"
+        url: "https://subirats.cat/adfsubirats/"
+        logo: "../img/logo-adf-subirats.png"
```
Comment (frgfm) [inline _config.yml:139]: This is not the same website nor the right logo. Why did you change it if I may?

File: _config.yml [hunk 2]
```diff
@@ -272,8 +272,8 @@ t:
         content: "The Chilean National Center of Artificial Intelligence with whom we are working for our first deployment abroad"
       partner4:
         title: "ADF Subirats"
-        url: "https://sfadf.org/adf/subirats/"
-        logo: "https://sfadf.org/wp-content/themes/sfadf/img/logo-header.jpg"
+        url: "https://subirats.cat/adfsubirats/"
+        logo: "../img/logo-adf-subirats.png"
```
Comment (frgfm) [inline _config.yml:276]: same here

File: _config.yml [hunk 3]
```diff
@@ -316,3 +316,271 @@ t:
     contact:
       title: "Follow the Pyronear adventure"
       content: "To stay updated about our progress or to join the adventure, you can also reach us on social media!"
+  ca:
+    label: ca
+    icon: 🇨🇦
```
Comment (frgfm) [inline _config.yml:321]: This is the canadian flag, not the catalan/valencian flag

File: img/logo-adf-subirats.png
Comment (frgfm) [inline img/logo-adf-subirats.png:1]: Could you point out where the logo comes from please? I don't see how it's related to the topic of the website :thinking:

**Review by frgfm — CHANGES_REQUESTED** [review]
Hey there :)
Thanks for the PR and translating! First off, I don't speak catalan so I can't check that part of the PR. Also, high-level question: I fully understand how it's relevant to add spanish, and perhaps even portuguese. However, the languages we first add need to be well picked: considering the situation in south america, adding portuguese on top of spanish makes total sense. And I imagine a visitor understands clearly how the languages are linked to the most impacted regions of the globe. I'm not against it, but I'd be curious to hear why you added catalan in that regards?

Apart from that, I added some comments, let me know what you think!

File: img/logo-adf-subirats.png
Comment (psd24) [reply img/logo-adf-subirats.png:1]: The logo that exists now does not correspond to the logo that our organization Adf Subirats has, which manages the cameras that we are managing.

File: _config.yml (hunk 1 above)
Comment (psd24) [reply _config.yml:139]: I changed it because what is now on the web is not correct. The logo and the link that have been uploaded now are those that represent our organization and take advantage of this PR to change them.

File: _config.yml (hunk 3 above)
Comment (psd24) [reply _config.yml:321]: There is an error here, I had not seen what the country's flag represents. Where can I find the flag of Catalonia?

**Comment by psd24** [comment]
> Hey there :) Thanks for the PR and translating! First off, I don't speak catalan so I can't check that part of the PR. Also, high-level question: I fully understand how it's relevant to add spanish, and perhaps even portuguese. However, the languages we first add need to be well picked: considering the situation in south america, adding portuguese on top of spanish makes total sense. And I imagine a visitor understands clearly how the languages are linked to the most impacted regions of the globe. I'm not against it, but I'd be curious to hear why you added catalan in that regards?
> 
> Apart from that, I added some comments, let me know what you think!

Add the Catalan language because we are the organization that has donated the cameras in order to improve the algorithm. We like being able to contribute to this project and promote it in our territory and thus be able to do things together.

I am with you that we would have to add the Portuguese language. Also give support in South America.

If you think that the Catalan language is not necessary, I would understand it and it can be removed.

File: _config.yml (hunk 3 above)
Comment (frgfm) [reply _config.yml:321]: If it can be found, I imagine it'll be among the unicode emoji : https://unicode.org/emoji/charts/full-emoji-list.html

File: _config.yml (hunk 1 above)
Comment (frgfm) [reply _config.yml:139]: Oh, perhaps adding a description would help then, because I had no context about this PR :sweat_smile:

File: img/logo-adf-subirats.png
Comment (frgfm) [reply img/logo-adf-subirats.png:1]: Alright, but is it not available online somewhere?

File: _config.yml (hunk 3 above)
Comment (psd24) [reply _config.yml:321]: I have been checking the emojis and the Catalan flag is not regulated by the Unicode Consortium, is there an alternative?

Comment (psd24) [reply _config.yml:321]: @frgfm @MateoLostanlen Do you know any solution to be able to add the Catalan flag? It's the only thing left to fix. thanks!

Comment (MateoLostanlen) [reply _config.yml:321]: @psd24 it seems available [here](https://iconduck.com/emojis/47135/catalonia-flag)

**Review by MateoLostanlen — APPROVED** [review]
Great job, thanks a lot for this PR @psd24,

I'll have to ask for the images to be translated too, but it's great as it is.

File: img/logo-adf-subirats.png
Comment (MateoLostanlen) [reply img/logo-adf-subirats.png:1]: it doesn't look like it

**Comment by MateoLostanlen** [comment]
@frgfm  All the changes have been made and the logo is not available online, is that ok with you?
